content/ri/using-redisinsight/auto-discover-databases.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: Auto discover databases
+weight: 10
+categories: ["RI"]
+path: auto-discover-databases/
+---
+This page lists the steps to be followed in order to automatically add on-premise Redis Enterprise and Redis Cloud databases.
```

Comment:
Redis Cloud Pro

Reply:
Would it be better if I place Pro in brackets?
Like this `..... and Redis Enterprise Cloud ( Pro ) databases`  
Also, place a note below it like : 
![Doc-Pro](https://user-images.githubusercontent.com/48902477/77914201-301ada00-72b3-11ea-8a6b-b4d74f295f5e.png)